examples/example05-parsing.cc

```diff
@@ -29,20 +29,13 @@ int main(int argc, char *argv[])
         parsed = Mata::Parser::parse_mf(fs, true);
         fs.close();
 
-        if (parsed.size() != 1) {
-            throw std::runtime_error(
-                    "The number of sections in the input file is not 1\n");
-        }
-        if (parsed[0].type.find("NFA") == std::string::npos) {
-            throw std::runtime_error("The type of input automaton is not NFA\n");
-        }
-
-        std::vector<Mata::IntermediateAut> inter_aut = Mata::IntermediateAut::parse_from_mf(parsed);
-
-        if (inter_aut[0].is_nfa())
-            construct(&aut, inter_aut[0]);
-    }
+        std::vector<Mata::IntermediateAut> inter_auts = Mata::IntermediateAut::parse_from_mf(parsed);
+        for (const auto& ia : inter_auts)
+            std::cout << ia << '\n';
```

Comment:
Is this write to `std::cout` intentional?

Reply:
Actually, it is intenional since it is in an example file. Do you have any suggestions how to do it better?

Reply:
You are right that since it is an example (I have not noticed that while doing the review), you probably want the user to run the example and see the generated output themself. There is no better solution that I know of how to show the user the required results. I consider this issue resolved, then.